designs/2022-community-eslint-org/README.md

```diff
@@ -0,0 +1,263 @@
+  currently maintained by [@fregante](https://github.com/fregante)  
+  <https://eslint.org/docs/8.0.0/user-guide/migrating-to-8.0.0#-removed-codeframe-and-table-formatters>
+- `eslint-plugin-eslint-plugin`  
+  This plugin is (together with `eslint-plugin-node`) a recommended plugin for
+  linting custom plugins.  
+  <https://eslint.org/docs/developer-guide/working-with-plugins#linting>
+
+## Documentation
+
+<!--
+    How will this RFC be documented? Does it need a formal announcement
+    on the ESLint blog to explain the motivation?
+-->
+
+I don't think this RFC needs a formal announcement on the ESLint blog.
+
+## Drawbacks
+
+<!--
+    Why should we *not* do this? Consider why adding this into ESLint
+    might not benefit the project or the community. Attempt to think
+    about any opposing viewpoints that reviewers might bring up.
+
+    Any change has potential downsides, including increased maintenance
+    burden, incompatibility with other tools, breaking existing user
+    experience, etc. Try to identify as many potential problems with
+    implementing this RFC as possible.
+-->
+
+I can't think of a reason why we would not want this.  
+Once we have access to the `@eslint-community` org & have the repos transfered,
+the "core" team can work out the rest of the process themselves.
+
+## Backwards Compatibility Analysis
+
+<!--
+    How does this change affect existing ESLint users? Will any behavior
+    change for them? If so, how are you going to minimize the disruption
+    to existing users?
+-->
+
+It won't affect existing ESLint users, as package could be released under the
+same name on `npm`.
+
+## Alternatives
+
+<!--
+    What other designs did you consider? Why did you decide against those?
+
+    This section should also include prior art, such as whether similar
+    projects have already implemented a similar feature.
+-->
+
+Creating a fork of the above mentioned packages. This could however create
+ceveral forks (which would cause fragmentation in the community), but also has
+the burden of needing to update all existing dependants.
+
+## Open Questions
+
+<!--
+    This section is optional, but is suggested for a first draft.
+
+    What parts of this proposal are you unclear about? What do you
+    need to know before you can finalize this RFC?
+
+    List the questions that you'd like reviewers to focus on. When
+    you've received the answers and updated the design to reflect them,
+    you can remove this section.
+-->
+
+- What exact acceptance criteria do we want to use?
+- What's the main place of communication for the "core" team?
+
+## Help Needed
+
+<!--
+    This section is optional.
+
+    Are you able to implement this RFC on your own? If not, what kind
+    of help would you need from the team?
+-->
+
+We would need to get access to the `@eslint-community` & be brought in contact
+with `@mysticatea` to transfer his repos to the new org & get `npm` publish
+rights.
```

Comment:
As said before: I currently own the `@eslint-community` npm org, but I'm happy to transfer it over to the ESLint team if preferred.

Also: I want to mention `@mysticatea` somewhere so that it's clear we can't do anything with the original repos otherwise, as contacting him ourselves didn't work obviously.

Reply:
> Also: I want to mention `@mysticatea` somewhere so that it's clear we can't do anything with the original repos otherwise, as contacting him ourselves didn't work obviously.

@ota-meshi actually mentioned that he did get some contact: https://github.com/eslint/rfcs/pull/91#issuecomment-1141987023

Also: The  `eslint-plugin-promise` can be moved anyway, so maybe it can be the first repo